composer.json

```diff
@@ -13,17 +13,18 @@
     "guzzlehttp/guzzle": "^6.3",
     "guzzlehttp/psr7": "^1.0",
     "psr/log": "^1.0",
-    "cache/cache": "2.0.x-dev#83fabc5bad8b90f7981c97898345df8935b0bf55 as 1.0",
     "symfony/serializer": "^3.4 || ^4.4",
     "symfony/property-access": "^3.4 || ^4.4",
     "doctrine/annotations": "^1.2",
     "symfony/property-info": "^3.4 || ^4.4",
     "symfony/lock": "^3.4 || ^4.4",
     "symfony/finder": "^3.0 || ^4.0 || ^5.0",
-    "phpdocumentor/reflection-docblock": "^5.2"
+    "phpdocumentor/reflection-docblock": "^5.2",
+    "cache/array-adapter": "^1.0",
```

Comment:
Oh, this is great, I like that a lot. I wonder why we didn't do this originally. I too can only find uses of `Cache\Adapter\PHPArray\ArrayCachePool`. Not seeing any other adapters in use in he package. Any recollections @deviantintegral ?

Reply:
I think I just wasn't aware that split packages were provided. TIL'ed!